elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Table.java

```diff
@@ -55,17 +57,21 @@
     @ToString.Exclude
     private Set<Dimension> dimensions;
 
+    @ToString.Exclude
+    private Set<String> tableTags;
+
     @Exclude
     @ToString.Exclude
     private final Map<String, Column> columnMap;
 
     public Table(Class<?> cls, EntityDictionary dictionary) {
-        if (!dictionary.getBindings().contains(cls)) {
+        if (!dictionary.getBoundClasses().contains(cls)) {
```

Comment:
If Table is going to be a simple POJO - we probably don't need to pass dictionary to it.  This will also simplify the Fili integration.

Reply:
If using builder, there is not need to provide a dictionary.